src/ppl/src/IOPlacer.cpp

```diff
@@ -1201,10 +1201,21 @@ void IOPlacer::placePin(odb::dbBTerm* bterm,
   Point lb = die_boundary.ll();
   Point ub = die_boundary.ur();
 
+  odb::dbTechLayer* tech_layer = tech_->findRoutingLayer(layer);
+  if (width < tech_layer->getWidth() ||
+      height < tech_layer->getWidth()) {
+    logger_->error(PPL,
+                   34,
+                   "Pin {} has dimension {}u which is less than the min width {}u of layer {}.",
+                   bterm->getName(),
+                   dbuToMicrons(std::min(width, height)),
```

Comment:
I would have done the min above and shared it with the if condition but that's just a minor point.

Reply:
Good point, I'll make this change for clarity in the code.